BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -363,22 +363,24 @@ class CbrainTask::<%= name %> < <%= (descriptor['custom'] || {})['cbrain:inherit
 % unless (descriptor['custom'] || {})['cbrain:ignore-exit-status']
     # Make sure <%= name %> completed successfully by checking its exit status
     # in +exit_cluster_filename+.
-    unless File.exists?(exit_cluster_filename)
-      self.addlog("Missing exit status file #{exit_cluster_filename}")
+    if File.exists?(exit_cluster_filename)
+      exit_status = IO.read(exit_cluster_filename).strip.to_i
```

Comment:
@prioux and @glatard what is your suggestion about the `"".to_i`, it should return an error exit code (which one?) instead of 0 ?